plugins/library/messagehub.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python
+import stomp
+import logging
+import logging.config
+from ansible.module_utils.basic import *
+
+
```

Comment:
Could you please follow: http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html